ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/sno/provision/cucushift-installer-rehearse-ibmcloud-ipi-sno-provision-chain.yaml

```diff
@@ -0,0 +1,22 @@
+chain:
+  as: cucushift-installer-rehearse-ibmcloud-ipi-sno-provision
+  steps:
+  - chain: ipi-conf-ibmcloud
+  - ref: ipi-install-monitoringpvc
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  - ref: cucushift-installer-check-sno  
+  - chain: cucushift-installer-check
+  - ref: cucushift-installer-check-ibmcloud
+  env:
+  - name: WORKERS
```

Comment:
If `SIZE_VARIANT` is set to `compact`, then we do not need to set `WORKERS` here, right ?